packages/flutter_tools/lib/src/commands/create.dart

```diff
@@ -663,11 +656,6 @@ https://flutter.dev/docs/development/packages-and-plugins/developing-packages#pl
       generatedCount += _injectGradleWrapper(project);
     }
 
-    if (boolArg('with-driver-test')) {
-      final Directory testDirectory = directory.childDirectory('test_driver');
-      generatedCount += await _renderTemplate('driver', testDirectory, templateContext, overwrite: overwrite);
-    }
```

Comment:
WDYT of leaving this in with some kind of warning message directing people to use `package:integration_test` instead and a link to its readme? There are still blog posts out there that advise using this.

Reply:
(I'd like to avoid triaging bugs where someone says "I read Blog/Doc x, and tried to use this command, and it doesn't work anymore)

Reply:
Sure! Do you want me to leave the templates in or just print a warning?

Reply:
I think just a warning is fine

Reply:
Done, PTAL